.github/workflows/default.yml

```diff
@@ -0,0 +1,121 @@
+name: default
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Cache pip packages
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key: lint-flake8-${{ runner.os }}-pip-${{ matrix.python-version }}-${{ hashFiles('setup.cfg') }}
+        restore-keys: |
+          lint-flake8-${{ runner.os }}-pip-${{ matrix.python-version }}
+          lint-flake8-${{ runner.os }}-pip-
+    - name: Install dependencies
+      env:
+        REQUIREMENTS_FILE: lint
+      run: |
+        python -m pip install -U pip setuptools
```

Comment:
Let's add `wheel` to this install list here and in all other occurrences as well.